tests/data/data.json

```diff
@@ -4,14 +4,24 @@
     "pid": "org1",
     "name": "The district of Martigny Libraries",
     "address": "Ave de la gare, Martigny CH-1920",
-    "code": "org1"
+    "code": "org1",
+    "default_currency": "CHF"
   },
   "org2": {
     "$schema": "https://ils.rero.ch/schema/organisations/organisation-v0.0.1.json",
     "pid": "org2",
     "name": "The district of Sion Libraries",
     "address": "Ave de la gare, Martigny CH-1950",
-    "code": "org2"
+    "code": "org2",
+    "default_currency": "CHF"
+  },
+  "org3": {
+    "$schema": "https://ils.rero.ch/schema/organisations/organisation-v0.0.1.json",
+    "pid": "org3",
+    "name": "Belgium beer librairies",
+    "address": "rue Lindemans 20, Bruxelles BE-1000",
+    "code": "org3",
+    "default_currency": "EUR"
```

Comment:
Is it **EUR** or **€**?

Reply:
I used the Currency ISO code (alpha_3 aka Ccy) (https://www.currency-iso.org/dam/downloads/lists/list_one.xml).
Using `pycountry.currencies` we can retrieve the currency label based on this alpha_3 code.